crates/runestick-macros/src/context.rs

```diff
@@ -521,7 +521,7 @@ impl Context {
             }
 
             impl #install_into_trait for #ident {
-                fn install_with(module: &mut #module) -> Result<(), #context_error> {
+                fn install_with(module: &mut #module) -> std::result::Result<(), #context_error> {
```

Comment:
Use `::std::result::Result`. It would guard against the case where someone defines their own local module called `std` as well!

Reply:
Yep makes sense